Justfile

```diff
@@ -9,7 +9,18 @@ target := if debug == '1' { 'debug' } else { 'release' }
 vendor_args := if vendor == '1' { '--frozen --offline' } else { '' }
 debug_args := if debug == '1' { '' } else { '--release' }
 cargo_args := vendor_args + ' ' + debug_args
-xdp_cosmic := '/usr/libexec/xdg-desktop-portal-cosmic'
+
+xdp_cosmic := `
+if [ -f "/usr/libexec/xdg-desktop-portal-cosmic" ]; then
```

Comment:
This requires COSMIC to be installed on the system doing the build, which is almost never the case (build systems are minimal and only contain build dependencies).

Reply:
hopefully those changes work better